src/nni_manager/config/adl/adaptdl-crd-v1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "apiVersion": "apiextensions.k8s.io/v1beta1",
```

Comment:
file name is `adaptdl-crd-v1.json` while the content apiVersion is `v1beta1`?

Reply:
the `v1` in the filename intends to refer to the specific CRD version, meaning that this JSON config is compatible with `adaptdl.petuum.com:v1`, while `adaptdl.petuum.com:v1` compatible with `apiextensions.k8s.io/v1beta1`

https://github.com/microsoft/nni/blob/f73690055d0046dea93cde0605f7852d311cf5ff/src/nni_manager/config/adl/adaptdl-crd-v1.json#L3-L9